UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs

```diff
@@ -669,15 +682,19 @@ private void ServerCheckQueueIsFlying(ref MoveData entry, ref bool fudged, ref V
 			    .magnitude >
 			    0.75f) //Resets play location if too far away
 			{
+				//TODO Force tile push if close enough??
 				if ((transform.position - entry.LocalPosition.ToWorld(MatrixManager.Get(entry.MatrixID)))
 				    .magnitude >
 				    3f)
 				{
-					ResetLocationOnClients();
+					reset = true;
+					//ResetLocationOnClients();
 				}
 				else
 				{
-					ResetLocationOnClients(true);
+					resetSmooth = true;
+					reset = true;
+					//ResetLocationOnClients(true);
```

Comment:
remove commented out stuff or put a reason explaining why its commented out

Reply:
iin case it's needed in the future

Reply:
![hoarders_wide-934a22c800c9158a2ac096a6986d02e6e3530d03](https://github.com/unitystation/unitystation/assets/43683714/58e385dc-10ef-4f09-8e89-23e684ae9eac)
In case it's needed in the future

Reply:
xD

Reply:
just remove the commented code. We can always get it back if we need it in the future because we use git.